### ISSUE TITLE
When applying changes, I want to see the files that where updated.

### ISSUE THREAD

**Issue by Antoine-lb** [Thomas-McKanna/Files2Prompt#15]
Hello again 👋
When using the workflow I find myself constantly opening the "Source Control" tab to see what has changed. You never know what files the LLM decided to change.

I find it annoying to not know what files where modified. The Windsurf AI IDE is particularly bad at this, it changes stuff without any notification or anything.

Here are my sugestions on how to cope with this:
- Open the files that were modified 
- Show a notification with each file that was changed (maybe 1 notification per file, with a button to "open the file")
- In the "Files2Prompt" tab, highlight the changed files

Honestly, anything that could make it more obvious the modified files would be a massive improuvement.

As always, thanks for you work, this plugin is amazing.

Note: we had an answer from the influencer I pocked (https://www.reddit.com/r/sveltejs/comments/1h6tw3q/comment/m2kgtse/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button), hope he likes it as much as I do.

**Comment by Thomas-McKanna** [comment]
I'll aim to tackle this over the weekend.

**Comment by Thomas-McKanna** [comment]
@Antoine-lb I did make an initial attempt at this with the latest update of the extension. Let me know what you think.

**Comment by Antoine-lb** [comment]
Hello 👋 Happy christmas!
I have payed with it today, but not that much, so let me play with it more next week with real work.

But I can already say some things: it's a lot better, and I will use this for sure.

There are 2 main things that I'm trying to achieve with this feature:
1. find the file that was changed to see what changed (checking the diff view in VSCode)
2. make sure that the AI didn't change files I didn't want it to change

The main problem with the current feature is that there is too many clicks to see the files. I have to click the "Open all Files" to then have to select the files I want, the click to view them.

The UI that I was thinking of is something like this:
![c7180104-b5d1-4f86-8ca1-58a6f90267c9](https://github.com/user-attachments/assets/8da8ab5f-06c0-4cc8-970f-7c333b7a73fc)

- A single Toast per file changed, that stays for 30 seconds
- Open File button, and if possible a "Open Diff" button that directs to the diff view from VSCode (I don't know if that is possible, but it would be very cool).
- "Open all" simply opens all files changed, no questions asked

With this UI I can quickly see all of the modified files, to see if it matches what I was expecting, and click on the file I wish to see more.

But again, let me play more with it next week, it's already pretty cool the way it is. Thanks!

**Comment by Thomas-McKanna** [comment]
Hey @Antoine-lb - I have decided to stop maintenance of this repository. I discovered a tool called [Aider](https://github.com/Aider-AI/aider) a couple of weeks ago and I have found it to essentially be a better version of my VS Code extension.

If you have not tried it yet, I highly recommend giving it a shot. It gives fine-grained control over the context by letting you select which files to include. It also supports copy/paste mode, but the typical usage involves using an API key for an LLM (and it supports many).
